README.md

```diff
@@ -1,16 +1,171 @@
-# Title
+# Iterator Drill: Map
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+* Define how the `map()` method works
+* Demonstrate `map()` with `Array`s
+* Demonstrate `map()` with complex data structures
+
 
 ## Introduction
 
-## SWBAT 1
+Iterating over arrays is a very common task we will find ourselves
+performing as developers. Using `for` loops, can get tedious after
+a while. Rather than using `for` loops and nesting to process data
+in lists and collections, you can take advantage a method like `map()`
+to better organize your logic into building blocks of functions,
+and then chain them to create more readable and understandable
+implementations.
```

Comment:
I feel we could make this paragraph flow a little better. For example: "As developers, we find ourselves responsible for all sorts of common but tedious tasks, such as iterating over arrays. While a `for` loop or nesting in lists and collections will complete the task, we can take advantage of a method like `map()` to organize and optimize our logic into building blocks of functions, which we can then chain together to create more readable and understandable implementations." It's all there, but smoothing it out will help students slide into the lesson more easily.